src/playback/internal/playbackcontroller.cpp

```diff
@@ -72,6 +80,8 @@ void PlaybackController::init()
     sequencer()->positionChanged().onNotify(this, [this]() {
         if (configuration()->cursorType() == PlaybackCursorType::SMOOTH) {
             m_tickPlayed.send(currentTick());
+        } else if (currentTick() % 480 < 24) {
+            m_tickPlayed.send(currentTick());
```

Comment:
Yes, this is a kludgy fix, but I don't know what else to do, because to do properly requires knowing the current time signature (to find out the beat length), and that's not available at this point.
The < 24 is required because the ticks received don't reliably fall on 1/4 note boundaries (multiples of 480), but in all the tests I did they were within 24 ticks after such boundaries.

Reply:
I think I have a better idea now...the metronome track should fix this issue, just need to emit silent ticks...but the metronome track doesn't seem to be working either!